traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -285,41 +285,6 @@ def create_asn(self, data=None):
 		"""
 
 	@api_request('put', 'asns', ('2.0',))
-	def create_asn(self, data=None, query_params=None):
-		"""
-		Create ASN
-		:ref:`to-api-asns`
-		:param data: The ASN data to use to replace the identified ASN.
-		:type data: Dict[str, Any]
-		:param query_params: 'id' is a required parameter, identifying the ASN to replace.
-		:type query_params: Dict[str, int]
-		:rtype: Tuple[Union[Dict[str, Any], List[Dict[str, Any]]], requests.Response]
-		:raises: Union[LoginError, OperationError]
-		"""
-
-	@api_request('post', 'asns', ('2.0',))
-	def create_asn(self, query_params=None):
-		"""
-		Create ASN
-		:ref:`to-api-asns`
-		:param query_params: 'id' is a required parameter, identifying the ASN to delete.
-		:type query_params: Dict[str, int]
-		:rtype: Tuple[Union[Dict[str, Any], List[Dict[str, Any]]], requests.Response]
-		:raises: Union[LoginError, OperationError]
-		"""
-
-	@api_request('get', 'asns/{asn_id:d}', ('2.0',))
-	def get_asn_by_id(self, asn_id=None):
-		"""
-		Get ASN by ID
-		:ref:`to-api-asns-id`
-		:param asn_id: The ID of the ASN to retrieve
-		:type asn_id: int
-		:rtype: Tuple[Union[Dict[str, Any], List[Dict[str, Any]]], requests.Response]
-		:raises: Union[LoginError, OperationError]
-		"""
-
-	@api_request('put', 'asns/{asn_id:d}', ('2.0',))
 	def update_asn(self, asn_id=None, query_params=None):
```

Comment:
`asn_id` should no longer be used as a param for `update_asn` - I don't think it causes any formatting problems, but it also doesn't do anything; you have to use `query_params` to pass the ASN's ID.